README.md

```diff
@@ -7,6 +7,29 @@ An opinionated static site generator written in Rust.
 Documentation is available on [its site](https://www.getgutenberg.io/documentation/getting-started/installation/) or
 in the `docs/content` folder of the repository.
 
+## Comparisons with other static site generators
+
+|                          | Gutenberg | Cobalt | Hugo | Pelican |
+|--------------------------|-----------|--------|------|---------|
+| Single binary            |     ✔     |    ✔   |   ✔  |    ✕    |
+| Language                 |    Rust   |  Rust  |  Go  |  Python |
+| Syntax highlighting      |     ✔     |    ✔   |   ✔  |    ✔    |
+| Sass compilation         |     ✔     |    ✔   |   ✕  |    ✔    |
+| Assets co-location       |     ✔     |    ✕   |   ✔  |    ✔    |
+| i18n                     |     ✕     |    ✕   |   ✔  |    ✔    |
+| Image processing         |     ✕     |    ✕   |   ✔  |    ✔    |
+| Search                   |     ✕     |    ✕   |   ✕  |    ✔    |
+| Sane template engine     |     ✔     |    ✔   |  ✕✕✕ |    ✔    |
```

Comment:
What do you mean by "sane"?  Should this instead name the template syntax used with a link to it?

Reply:
Mostly whether the template engine will drive you insane when you try to make a nice design. That line is very subjective but I will keep it like that, it's the initial reason of Gutenberg existence after all :)